UrlShortener/shrtn.yml

```diff
@@ -1,3 +1,10 @@
+httpConnection:
+  # the hostname of our web server
+  host: localhost
+
+  # the port to use
+  port: 8080
```

Comment:
Since there is an official [property](http://www.dropwizard.io/0.9.3/docs/manual/configuration.html#man-configuration-http) that actually controls what port the application runs on, having a duplicate here only opens up room for error.

If you remove this to avoid duplication then it makes less sense to have an httpConnection.host hierarchy with only one element, so perhaps just having "hostname" at the top of Configuration is sufficient.

In a real world deployment, your app might have a custom port in the LAN but the load balancer would use standard HTTP (80 or HTTPS 443) to be friendly to the outside world.  In that case the "http://" or "https:" would be sufficient for selecting the port.